internal/resources/package/data_source_package.go

```diff
@@ -40,3 +50,111 @@ var packageSchema = map[string]*schema.Schema{
 	common.MetaKey:       common.Meta,
 	spec.SpecKey:         spec.SpecSchema,
 }
+
+func resourcePackageRead(ctx context.Context, d *schema.ResourceData, m interface{}) (diags diag.Diagnostics) {
```

Comment:
The changes look good, can you schedule a call to validate if it works as expected?

Reply:
updated